ruby_on_rails/linting_and_automatic_check.md

```diff
@@ -62,11 +62,12 @@ Use the [provided reek configuration](../templates/.reek.yml) by adding a `.reek
 
 ## SCSS lint
 
-```ruby
-group :development, :test do
-  gem 'scss_lint', require: false
-end
-```
+To lint the SASS/SCSS files in our project we are going to use the `sass-lint` npm package.
+You should have it installed in your machine, if not, you can do so by running `bin/yarn add global sass-lint`.
+Add to the project the linter configuration file you can find in the templates folder and check the `bin/fastcheck`
+template to see the command to execute the SCSS linting.
+
+When you add this linter, remember to add also `npm install -g sass-lint` in the list of setup commands on SemaphoreCI.
```

Comment:
instead of relying on the global package, why don't we install it locally and run it with `yarn sass-lint` ? this would also install it on semaphore automatically

Reply:
agree. I change it 👍